### DISCUSSION TITLE
О флибусте и "больших" архивах

### DISCUSSION THREAD

**Discussion by plotn** [petrovvlad/freeLib#55] [Ideas]
Доброго дня!
возникла у меня тут мысль. Вот флибуста - там куча архивов, при этом в них - куча книг. Программа умеет вполне нормально заходить в архивы, за это ей огромное спасибо.
Но мысль все же свербит - а нет ли таких ситуаций, когда в "большом" архиве лежат книги "fb2.zip", как в таком случае будет? Пройдется ли программа в 2 уровня архива? Не проверял, но так, на подумать. Может внести отдельную опцию "2 уровня архива" ? Понятно, что может захотеться и 3, но это можно уже считать перебором.

**Comment by petrovvlad** [comment]
Работа с вложенными архивами пока не предусмотренна. Я пока не встречал таких библиотек. Но если будет спрос на, то может  быть сделаю :)

**Comment by plotn** [comment]
ну, где ж его взять то, спрос ;)))) но всегда же приятно быть в чем то первым
